recipes/z5py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.5.3" %}
+# {% set checksum = "c644f3457b31be2fa0ce8fce15536e2eaeab160d" %}
+
+package:
+  name: z5py
+  version: {{ version }}
+
+source:
+    url: https://github.com/constantinpape/z5/archive/{{ version }}.tar.gz
+    # sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - boost 1.67.0
```

Comment:
To clarify the earlier comment, we can drop the explicit pinning. It will be applied for us automatically.

Reply:
Ok, got it.